docs/docs/authentication.md

```diff
@@ -4,98 +4,120 @@ description: Set up an authentication provider
 
 # Authentication
 
-`@redwoodjs/auth` contains both a built-in database-backed authentication system (dbAuth), as well as lightweight wrappers around popular SPA authentication libraries.
+Redwood has integrated auth end to end, from the web side to the api side.
+On the web side, the router can protect pages via the `Private` component, and even restrict access at the role-level.
+And if you'd prefer to work with the primitives, the `useAuth` hook exposes all the pieces to build the experience you want.
 
-- dbAuth _([Redwood Docs](https://redwoodjs.com/docs/auth/dbauth))_
+Likewise, the api side is locked down by default: all SDLs are generated with the `@requireAuth` directive, making sure that making something publicly available is something that you opt into rather than out of.
+You can also require auth anywhere in your Services, and even in your server-ful or -less functions.
 
-We currently support the following third-party authentication providers:
+Last but not least, Redwood provides it's own auth provider: [dbAuth](./auth/dbauth.md).
 
-- Auth0 _([Repo on GitHub](https://github.com/auth0/auth0-spa-js))_
-- Azure Active Directory _([Repo on GitHub](https://github.com/AzureAD/microsoft-authentication-library-for-js))_
-- Clerk _([Website](https://clerk.dev))_
-- Firebase _([Documentation Website](https://firebase.google.com/docs/auth))_
-- Magic Links - Magic.js _([Repo on GitHub](https://github.com/MagicHQ/magic-js))_
-- Netlify Identity _([Repo on GitHub](https://github.com/netlify/netlify-identity-widget))_
-- Netlify GoTrue-JS _([Repo on GitHub](https://github.com/netlify/gotrue-js))_
-- Nhost _([Documentation Website](https://docs.nhost.io/platform/authentication))_
-- Okta _([Repo on GitHub](https://github.com/okta/okta-auth-js))_
-- Supabase _([Documentation Website](https://supabase.io/docs/guides/auth))_
-- WalletConnect _([Repo on GitHub](https://github.com/oneclickdapp/ethereum-auth))_
+In this doc, we'll cover auth in Redwood at a high level.
+All auth providers share the same interface so the information here will be useful no matter which auth provider you use.
 
-You can also implement your own custom auth client. Check out the [Custom docs](auth/custom) for more info.
+## Official integrations
 
-:::info Auth Playground
+When you're ready to add auth to your app, there's a great list of providers to choose from, and if you don't see the one you like, you can integrate it via a simple API:
```

Comment:
I don't know the correct English word for it, but in Swedish we'd say that this sentence has a misdirection. It's where a part of the sentence "points to"/"references" the wrong thing. (Please tell me if you know the correct English word for this.)

To fix it, you have to switch things around like this

```suggestion
Redwood has a simple API to integrate any auth provider you can think of. But to make it easier for you official integrations for some of the most popular auth providers are provided out of the box:
```

Reply:
I was confused what you meant at first but I looked at what's after the colon and get what you mean. And if there's a word for it I'm not sure what it is. A misdirection of some kind sounds right. Maybe misplacing the emphasis?